### DISCUSSION TITLE
Dark mode / theme support

### DISCUSSION THREAD

**Discussion by kepano** [alexandru-dinu/obsidian-sortable#8] [Ideas]
Great work! I am trying to add support for Sortable in Minimal theme and I noticed that the icons do not show up in the DOM. I am wondering if there is a way to target those elements?

**Comment by alexandru-dinu** [comment]
Hello, Stephan!

The icons (up/down arrows) are set as the `background-image` for the `th` elements, in [styles.css](https://github.com/alexandru-dinu/obsidian-sortable/blob/master/styles.css). The arrows are [changed](https://github.com/alexandru-dinu/obsidian-sortable/blob/ab87ab11ab7fcd2070fc875189efdf2ae4b680ff/main.ts#L62) according to the class of the header (`header-sort-up`, `header-sort-down`).

Sorry if I am not understanding this right, but there is no DOM element corresponding to the arrows, it's just a CSS property of the table header.

**Reply by kepano** [reply]
I see it now, okay. 

One suggestion — your arrow icons are linked from an online source. To make the icons work in offline mode it would be preferable to use a data URI or embed them in the plugin

https://yoksel.github.io/url-encoder/

**Reply by alexandru-dinu** [reply]
Thank you for this!

I fixed the arrows in [v0.1.1](https://github.com/alexandru-dinu/obsidian-sortable/releases/tag/0.1.1). They are rendered from the corresponding URIs depending on `.theme-light` or `.theme-dark` class.
